string class_name
float32 speed
string direction
geometry_msgs/Point position
